docs/source/glossary.rst

```diff
@@ -50,8 +50,10 @@ Glossary
     being committed to.
 
   View
-    The period of time of PBFT when the current primary is in charge. The view
-    changes when the primary is deemed faulty (see
+    The period of time when the current primary node is in charge. The
+    view changes at regular intervals (controlled by the on-chain setting
+    ``sawtooth.consensus.pbft.view_change_duration``), and when the primary node
```

Comment:
It's actually the `sawtooth.consensus.pbft.forced_view_change_interval` setting that controls this. `view_change_duration` is something different.

Reply:
Fixed.